website/docs/r/cosmosdb_mongo_collection.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cosmosdb_mongo_collection"
+sidebar_current: "docs-azurerm-resource-cosmosdb-mongo-collection"
+description: |-
+  Manages a Mongo Collection within a Cosmos DB Account.
+---
+
+# azurerm_cosmosdb_mongo_collection
+
+Manages a Mongo Collection within a Cosmos DB Account.
+
+## Example Usage
+
+```hcl
+data "azurerm_cosmosdb_account" "example" {
+  name                = "tfex-cosmosdb-account"
+  resource_group_name = "tfex-cosmosdb-account-rg"
+}
+
+resource "azurerm_cosmosdb_mongo_database" "example" {
+  name                = "tfex-cosmos-mongo-db"
+  resource_group_name = "${data.azurerm_cosmosdb_account.example.resource_group_name}"
+  account_name        = "${data.azurerm_cosmosdb_account.example.name}"
+}
+resource "azurerm_cosmosdb_mongo_collection" "example" {
+  name                = "tfex-cosmos-mongo-db"
+  resource_group_name = "${data.azurerm_cosmosdb_account.example.resource_group_name}"
+  account_name        = "${data.azurerm_cosmosdb_account.example.name}"
+  database_name       = "${data.azurerm_cosmosdb_account.example.name}"
+  
+  default_ttl_seconds = "777"   
+  shard_key           = "uniqueKey"
+  
+  indexes {
+    key    = "aKey"
+    unique = false
+  }
+  
+  indexes {
+    key    = "uniqueKey"
+    unique = true
+  }
+}
+
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Cosmos DB Mongo Collection. Changing this forces a new resource to be created.
+* `resource_group_name` - (Required) The name of the resource group in which the Cosmos DB Mongo Collection is created. Changing this forces a new resource to be created.
+* `resource_group_name` - (Required) The name of the Cosmos DB Mongo Database in which the Cosmos DB Mongo Collection is created. Changing this forces a new resource to be created.
+* `default_ttl_seconds` - (Required) The default Time To Live in seconds. If the value is `-1` items are not automatically expired.
+* `shard_key` - (Required) The name of the key to partition on for sharding. There must not be any other unique index keys. 
+* `indexes` - (Optional) One or more `indexes` blocks as defined below.
+
+---
+
+An `indexes` block supports the following:
+
+* `key` - (Required) The name of the key to use for this index.
+* `unique` - (Required) Should the index key be unique?
```

Comment:
I've always found phrasing the description as a question to be awkward. Do you think 
```suggestion
* `unique` - (Required) Whether the index key should be unique.
```
sounds a little better?